LISISO/commonfunctions.sh

```diff
@@ -7,27 +7,29 @@
 GetDistroName()
 {
 	linuxString=$(grep -ihs "CentOS\|Red Hat Enterprise Linux" /etc/redhat-release)
-
-	case $linuxString in
-		*CentOS*)
-			distro_name=CentOS
-			;;
-		*Red*)
-			distro_name=RHEL
-			;;
-		*Oracle*)
-                        distro_name=Oracle
-                        ;;
-
-		*)
-			distro_name=unknown
-			return 1
-			;;
-	esac
-
+	OracleDistroName=$(grep -ihs "Oracle" /etc/oracle-release)
+	if [[ $OracleDistroName == *Oracle* ]]; then
+        	distro_name="Oracle"
+		distro_package_name=RPMS
```

Comment:
Do we need to set distro_package_name=RPMS in this function?

Reply:
yes. Then only we can enter in to corresponding RPMS67 or any  folder and install.

Reply:
if distro_package_name is set RPMS only , do we need to set it for different types of distros?

Reply:
we can place only once.